### PR TITLE
fix string freeze issue for ruby 2.3

### PR DESCRIPTION
**Pull request by alexandru-calinoiu**
I hit this issue in tests using fog-aws mock services.

### CODE REVIEW

File: spec/storage_spec.rb [hunk 1]
```diff
@@ -58,7 +64,9 @@ module BothWays
     module Fog
       module BothWays
         class Storage
-          def initialize(_args); end
+          def initialize(_args)
+            ;
```
Comment (plribeiro3000) [inline spec/storage_spec.rb]: Why did you change this?

**Comment by plribeiro3000** [comment]
@alexandru-calinoiu Sounds awesome. Would you squash the changes please?

Thanks!

**Comment by alexandru-calinoiu** [comment]
@plribeiro3000 done.

**Comment by plribeiro3000** [comment]
Awesome. Thx!

**Comment by geemus** [comment]
Thanks!
